src/main.ts

```diff
@@ -61,16 +63,17 @@ async function run(): Promise<void> {
       '/var/run/docker.sock:/var/run/docker.sock'
     ]
 
-    if (fs.existsSync(configFile)) {
-      commandOptions.push(
-        '--mount',
-        `type=bind,source=${configFile},target=/.dive-ci`,
-        '--ci-config',
-        '/.dive-ci'
-      )
+    const cmdOptions = []
+    
+    if (configFile) {
+      runOptions.push('-v', `${configFile}:/.dive-ci`)
+      cmdOptions.push('--config-file', '/.dive-ci')
```

Comment:
Should this not use the yaml file supplied in the parameters?

Reply:
Nevermind. I see that the config file is mapped to this name inside the container..